website/src/pages/roadmap.md

```diff
@@ -8,64 +8,68 @@ Hudi community strives to deliver major releases every 3-4 months, while offerin
 This page captures the forward-looking roadmap of ongoing & upcoming projects and when they are expected to land, broken
 down by areas on our [stack](blog/2021/07/21/streaming-data-lake-platform/#hudi-stack).
 
+## Recent Release
+[0.15.0](https://issues.apache.org/jira/projects/HUDI/versions/12353381) (June 2024)
+
 ## Future Releases
 
-Next upcoming release : [0.14.1](https://issues.apache.org/jira/projects/HUDI/versions/12353493) (Dec 2023)
+- [1.0.0-beta2](https://issues.apache.org/jira/projects/HUDI/versions/12354810) (July 2024)
+- [0.16.0](https://issues.apache.org/jira/projects/HUDI/versions/12354773) - Bridge release supporting reads of both 1.x and 0.x Hudi versions. (Q3, 2024)
+- [1.0.0](https://issues.apache.org/jira/projects/HUDI/versions/12353848) (Q3, 2024)
```

Comment:
These links are not publicly accessible somehow.  Could you check them again or remove them?

Reply:
Looks like they are accessible only after signing in. Let me remove them.